### PR TITLE
Angular 2

### PR DESCRIPTION
**Pull request by radustan**

### CODE REVIEW

File: ang2/app/app.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        'App1.AboutController',
+        'App1.ContactController',
+        'App1.hcAreaChart',
+        'App1.callsService'
+    ]
+);
+
+app.config(function($routeProvider) {
+    $routeProvider
+    // route for the home page
```
Comment (gabrielaraducan) [inline ang2/app/app.js]: indentation :D

File: ang2/index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                </div>
+                <ul class="nav navbar-nav navbar-right">
+                    <li ng-repeat="item in menu" ng-class="{ active: isActive('/{{item.name}}')}">
+                        <a href="#{{item.name}}">{{item.label}}</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+<div class="container theme-showcase" role="main">
```
Comment (strajeru) [inline ang2/index.html]: Indentation please.

File: ang2/phpApp/chartData.php [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        ], [
+            'name'=> 'America',
+            'data'=> [18, 31, 54, 156, 339, 818, 1201]
+        ], [
+            'name'=> 'Oceania',
+            'data'=> [2, 2, 2, 6, 13, 30, 46]
+        ]]
+);
+
+$params = $_GET;
```
Comment (strajeru) [inline ang2/phpApp/chartData.php:23]: is this needed?

**Pull request merged**